doc/DeveloperGuide.md

```diff
@@ -94,6 +94,10 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should be able to be installed easily by non-technical user
+6. Should be able to run with minimal RAM of 2GB
+7. Should be able to export the address book to other system
```

Comment:
A `export the addressbook` seems to be in the grey area between NFR and FR, perhaps you can consider generalizing the case.